dom.bs

```diff
@@ -4355,6 +4361,7 @@ method must run these steps:
 
    <dt>{{DocumentType}}
    <dt>{{DocumentFragment}}
+   <dt>{{Attr}}
```

Comment:
This one would be "Return the result of locating a namespace prefix for its element, if that is not null, and null otherwise."